airflow/contrib/operators/dataproc_operator.py

```diff
@@ -769,7 +769,7 @@ def __init__(
         self.dataproc_properties = dataproc_pig_properties
         self.dataproc_jars = dataproc_pig_jars
         self.region = region
-        self.job_error_states = job_error_states
+        self.job_error_states = job_error_states if job_error_states is not None else ['ERROR']
```

Comment:
Nice one. While we're at it, how about making this a set as well? I expect to error states to be unique.

Reply:
Good catch. I don't see any reason for duplicates in it, so set is better.

I scanned through `airflow.contrib.hooks.gcp_dataproc_hook.py` and there's nothing requiring/checking list type, so I don't see any problems and will replace all `job_error_states` by sets.

Reply:
@Fokko I pushed the changes, CI is rerunning.